<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiTetris</class>
 <widget class="QWidget" name="GuiTetris">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="TetrisBoard" name="mp_board">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>231</width>
     <height>322</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mp_labelNextPieceString">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>9</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Piece</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mp_labelNewPiece">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>28</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mp_pbStart">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mp_pbPause">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TetrisBoard</class>
   <extends>QFrame</extends>
   <header>TetrisBoard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
